### DISCUSSION TITLE
BTV Comedy

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#474] [Show and tell]
### Channel ID

bTVComedy.bg

### Stream URL (optional)

https://cdn3.gledam.xyz/hls/btv-comedy/index.m3u8?e=1692278211&hash=90gpfKQEq98mVdSSDv8QOA

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
Offline.
<img width="734" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/19af754c-e0e2-43ad-8dae-57f6a843c7a6">
